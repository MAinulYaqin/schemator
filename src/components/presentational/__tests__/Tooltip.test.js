import React from "react";
import { shallow } from "enzyme";

import Tooltip from "../Tooltip";

describe("Tooltip", () => {
  const setup = propOverrides => {
    const props = {
      text: "My Tooltip",
      ...propOverrides
    };

    const wrapper = shallow(<Tooltip {...props} />);

    return {
      wrapper,
      props
    };
  };

  it("should renders default", () => {
    const { wrapper } = setup();

    expect(wrapper).toMatchSnapshot();
  });

  it("should renders with addtional class", () => {
    const { wrapper } = setup({ className: "my-class" });

    expect(wrapper).toMatchSnapshot();
  });
});
