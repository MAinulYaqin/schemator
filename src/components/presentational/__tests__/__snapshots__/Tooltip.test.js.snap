// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip should renders default 1`] = `
<styled.div>
  <styled.div />
  <styled.span>
    My Tooltip
  </styled.span>
</styled.div>
`;

exports[`Tooltip should renders with addtional class 1`] = `
<styled.div
  className="my-class"
>
  <styled.div />
  <styled.span>
    My Tooltip
  </styled.span>
</styled.div>
`;
